### ISSUE TITLE
Finetuning Stable Diffusion (InstructPix2Pix) of amodal completion

### ISSUE THREAD

**Issue by huangpeng6666** [AndreeaDogaru/Gen3DSR#7]
Nice work!
Could you share the data preparation and codes of finetuning Stable Diffusion (InstructPix2Pix), for amodal completion task.
Thank you very much.

**Comment by AndreeaDogaru** [comment]
Hello, 

Thank you for your interest in our work! Currently, I'm on an internship and will only be able to share additional resources starting with March 2025. However, I can recommend you this [tutorial](https://github.com/huggingface/diffusers/blob/main/examples/instruct_pix2pix/train_instruct_pix2pix.py) for training InstructPix2Pix.  We create our training dataset starting from the Blender-rendered images of the objects in [OmniObject3D](https://openxlab.org.cn/datasets/OpenXDLab/OmniObject3D-New/tree/main/raw/blender_renders). To obtain the conditioning view, we mask-out parts of the object by randomly overlaying the silhouette of an arbitrary object sampled from the [Objaverse](https://github.com/cvlab-columbia/zero123#dataset-objaverse-renderings) dataset. The category labels provided in OmniObject3D are used as prompts to guide the diffusion process. We train the model at a resolution of 512x512 for 25000 iterations with a batch size of 16.

**Comment by huangpeng6666** [comment]
Thanks for your reply!
